kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/JbpmKieServerExtension.java

```diff
@@ -787,6 +790,37 @@ protected EntityManagerFactory build(Map<String, String> properties) {
         }
     }
 
+    private void checkAndAddTaskAssigningEntities(PersistenceUnitInfo info) {
+        if ("false".equals(System.getProperty(KIE_TASK_ASSIGNING_RUNTIME_EXT_DISABLED))) {
```

Comment:
It's more readable to invert the logic, so that you just:
`if (true.equals(...)) { return; }` and the rest of the method keeps the original level of indentation.

Reply:
Inverting the condition would make it more complex because you'd have to `return` if the property was "true" or unset.

The way it's written now means the task assigning extension is disabled by default and must be enabled explicitly (that's probably wanted). Wouldn't be more natural to enable it like this:
```
-Dorg.kie.server.taskAssigning.runtime.ext.enabled=true
```
rather than
```
-Dorg.kie.server.taskAssigning.runtime.ext.disabled=false
```
The double negative is a bit difficult to understand. It makes better sense to me to use a "*disabled" property to disable an extension that is enabled by default and, conversely, use a "*enabled" property to enable an extension that is disabled by default.

Also I'd suggest to use `Boolean.parseBoolean()` consistently because it's used on other places to parse properties. `Boolean.parseBoolean()` is case insensitive and so while `~~~.disabled=FALSE` might work for other properties elsewhere, it wouldn't work here, which would be surprising.

Reply:
The standard mechanism for enabling disabling extensions is
org.kie.server.XXX.XXX.ext.disabled so I'd prefer to keep it to this extension too. Otherwise it's not straight ahead to understand how to enable/disable a particular extension.
So I basically vote for keeping the standard nomeclature
      org.kie.server.taskAssigning.runtime.ext.disabled

And finally with the code

"false".equals(System.getProperty(KIE_TASK_ASSIGNING_RUNTIME_EXT_DISABLED)

Since this is not let's say not a standard extension, I want be sure to enable it exactly when the user typed the word "false".
Otherwise if we do Boolean.parse("truee") we could enable the extension just because of type mistake.

Reply:
OK, I understand your reasons for using the `disabled=false` form to activate the extension and agree with your approach.